Code/src/main/java/com/html5parser/insertionModes/Initial.java

```diff
@@ -90,6 +91,11 @@ public void anythingElse(ParserContext parserContext) {
 		// parse error; set the Document to quirks mode.
 		// In any case, switch the insertion mode to "before html", then
 		// reprocess the token.
+		Document doc = parserContext.getDocument();
+		if (doc.getElementsByTagName("iframe").getLength() == 0) {
+			parserContext.addParseErrors(ParseErrorType.UnexpectedToken);
+			parserContext.setFlagForceQuirks(true);
```

Comment:
This flag force quirks  is not the same as setting the document to quirks mode. This is wrong. Does actually fix a test case?? @Prin4